packages/cascara/src/ui/Table/mockData/index.js

```diff
@@ -0,0 +1,23 @@
+import faker from 'faker';
+
+faker.seed(1);
+
+export const generateFakeEmployees = (qty) =>
+  new Array(qty).fill(null).map((e, i) => {
+    // So we never have collisions, we have to start at 2 instead of 1
+    faker.seed(i++ + 1);
+    return (e = {
+      avatar: faker.image.avatar(),
+      country: faker.address.countryCode(),
+      date: faker.date.future(),
+      department: faker.name.jobArea(),
+      eid: faker.random.uuid(),
+      fullName: faker.fake(
+        '{{name.lastName}}, {{name.firstName}} {{name.suffix}}'
+      ),
+      homePhone: faker.phone.phoneNumber(),
+      nickname: faker.name.firstName(),
+      officePhone: faker.phone.phoneNumber(),
+      title: faker.name.jobTitle(),
+    });
+  });
```

Comment:
I love that you went ahead and did this BTW. We should talk in a month and see what other needs we might have for generating consistently seeded, random data. There are going to be other needs we have too. Might be good to invest a little more in some good tooling for faster prototyping.